src/components/Discussion/Internal/Composer/Actions.js

```diff
@@ -16,24 +17,24 @@ const actionButtonStyle = {
   alignSelf: 'stretch',
   display: 'flex',
   alignItems: 'center',
-  height: '40px',
-  lineHeight: '40px',
+  minHeight: pxToRem(40),
+  lineHeight: pxToRem(40),
   padding: '0'
 }
 
 const styles = {
   root: css({
     display: 'flex',
+    flexFlow: 'wrap',
     justifyContent: 'space-between'
   }),
   mainActions: css({
-    marginLeft: 'auto',
-    display: 'flex'
+    display: 'flex',
+    overflow: 'hidden'
```

Comment:
why is this overflow hidden necessary? Could we remove it and prevent cut offs? I'd prefer slightly broken layout over cutoffs.